### PR TITLE
Issue 3910 - take three.

### PR DESCRIPTION
**Pull request by 48klocs**
We can have objectives on uninstanced items, they just might be tied to the hash.

#3910 (again).

wendigogogogogo

### CODE REVIEW

File: src/app/inventory/store-types.ts [hunk 1]
```diff
@@ -106,6 +107,10 @@ export interface DimStore {
     progressions: DestinyProgression[];
   };
 
+  uninstancedItemObjectives: null | {
```
Comment (bhollis) [inline src/app/inventory/store-types.ts]: I don’t think you need to add these to the store just to pass them down to the function that makes items. Better to just pass this new map down as a new argument.

**Comment by bhollis** [comment]
There seems to be another issue - in your example screenshot, the item is marked as complete (yellow border) when it’s not. You’ll need to take a followup look at how complete is calculated. It seems like the completion percentage is still correct...

**Comment by delphiactual** [comment]
@bhollis that is the item above it in `bounties`

**Comment by bhollis** [comment]
Got it, I mistook what the popup was attached to (makes me think I should do something to fix that). In that case go ahead, the weird plumbing of the values can ve fixed later if you don’t want to do it now.

**Pull request merged**